Seatbelt/Commands/Products/OfficeMRUsCommand.cs

```diff
@@ -33,16 +38,286 @@ public OfficeMRUsCommand(Runtime runtime) : base(runtime)
         }
 
 
+        private void SetRegistryKey(string application, string keyVal)
+        {
+            for (int i = 7; i <= 16; i++)
+            {
+                string version = i.ToString() + ".0";
+                RegistryKey vbomKey = Registry.CurrentUser.OpenSubKey($@"SOFTWARE\Microsoft\Office\{version}\{application}\Security", true);
+                if (vbomKey != null)
+                {
+                    if (vbomKey.GetValueNames().Contains("AccessVBOM"))
+                    {
+                        string keyPath = vbomKey.ToString() + @"\AccessVBOM";
+                        WriteWarning($"Setting {keyPath} to {keyVal}");
+                        vbomKey.SetValue("AccessVBOM", keyVal, Microsoft.Win32.RegistryValueKind.DWord);
+                        vbomKey.Close();
+                    }
+                }
+            }
+        }
+
+        // Searches for blocking keys in HKCU and switches them if it's found
+        // Returns true if there is a key that blocks us
+        // Input is the office application. Currently 'Word|Excel'.
+        private bool CheckHKCURegistryBlock(string application)
+        {
+            bool blocked = false;
+            // Version reg keys found from Stack Overflow: https://stackoverflow.com/questions/3266675/how-to-detect-installed-version-of-ms-office
+            // Iterate over all possible versions looking for our security key called AccessVBOM
+            for (int i = 7; i <= 16; i++)
+            {
+                string version = i.ToString() + ".0";
+                RegistryKey vbomKey = Registry.CurrentUser.OpenSubKey($@"SOFTWARE\Microsoft\Office\{version}\{application}\Security", true);
+                if (vbomKey != null)
+                {
+                    if (vbomKey.GetValueNames().Contains("AccessVBOM"))
```

Comment:
On my test machine, the "AccessVBOM" key does not exist by default.  Can you create this value if it does not exist, and delete it later?

Reply:
Good point, overlooked that test. Should be fixed in 4df9e19e712e94bdc06c65fa0c8b3f184e56f533.